cernopendata/modules/fixtures/data/records/atlas-simulated-datasets.json

```diff
@@ -1,7 +1,7 @@
 [
   {
     "abstract": {
-      "description": "The ATLAS open data dataset is comprised of real data recorded with the ATLAS detector in 2012 and matching simulated data.\n      Both real and simulated data is subjected to a loose event preselection to reduce processing time by reducing the overall number of events that have to be analysed."
+      "description": "The ATLAS open data dataset is comprised of real data recorded with the ATLAS detector in 2012 and matching simulated data.\n      Both real and simulated data is subjected to a loose event preselection to reduce processing time by reducing the overall number of events that have to be analysed. <p>These simulated datasets correspond to the data collected by the ATLAS experiment in 2012.\n      </p>"
```

Comment:
Here's another situation: the original `description` does not contain `<p>..</p>` while the original `note` does.  Currently, the merged field contains `DESCRIPTION_TEXT <p> NOTE_TEXT</p>` in this PR.  It would be better to detect this situation and generate instead `<p>DESCRIPTION_TEXT</p><p> NOTE_TEXT</p>` so that the field markup is consistent.